runtime/src/accounts_index.rs

```diff
@@ -652,7 +652,7 @@ impl RollingBitField {
 
 #[derive(Debug)]
 pub struct RootsTracker {
-    roots: RollingBitField,
+    pub roots: RollingBitField,
```

Comment:
It looks like we only need
```suggestion
    pub(crate) roots: RollingBitField,
```